utils/cosmos.ts

```diff
@@ -332,3 +333,58 @@ export async function sendNFTsToAPIWallet (
 
   return transactionHash
 }
+
+export function amountToLIKE (likecoin: any, denom: string) {
+  if (!likecoin) { return -1 }
+  if (likecoin.denom === denom) {
+    return (new BigNumber(likecoin.amount)).dividedBy(1e9).toFixed()
```

Comment:
use shiftedBy instead?

Reply:
[🎨 Update amountToLIKE to use shiftedBy](https://github.com/likecoin/nft-book-press/pull/290/commits/0bf4766308d3a99f789f3ee52a789d1e2ee53cda)